ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -3,29 +3,28 @@
 #initializes sd card on rpi
 
 SDDIR="/dev/mmcblk0"
-SDPARTDIR="/dev/mmcblk0p1"
+SDPARTDIR="/dev/mmcblk0p3"
 
 if [ -b "$SDDIR" ]; then
   echo "SD Card found continuing..." | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` > /etc/rc.local.logs
   mkdir -p /media/plugin-data/ | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
   if [ ! -b "$SDPARTDIR" ]; then
         echo "Initializing Partition Table on SD Card" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
         sed -e 's/\s*\([\+0-9a-zA-Z]*\).*/\1/' << FDISK_CMDS | fdisk /dev/mmcblk0
-                o # clear the in memory partition table
-                g #initialize got partition table
                 n # new partition
-                1 # partition number 1
-                # default - start at beginning of disk
+                p # primary partition
+		3 # partition number 3
+                3637248 # default - start at beginning of free space of disk (after pi os)
```

Comment:
is there anyway to discover this value dynamically instead of hard-coding it?  I think you may be able to use a Debian tool to know where the last sector of the 2nd partition is.